pkgs/by-name/jn/jnv/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, autoconf
+, automake
+, clang
+, libclang
+, libtool
+}:
+rustPlatform.buildRustPackage rec
+{
+  pname = "jnv";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "ynqa";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LN42UyXOStwgj/bbHl2MK1jTsDACTjDo+QaoZ9FzurA=";
+  };
+
+  cargoHash = "sha256-ZCYDlvKSuEONPSz/TS00mYolPnMYxsOwE9a0vK36UQQ=";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    clang
+    libclang
+    libtool
+  ];
+
+  cargoPatches = [
+    ./cargo-lock.patch
+  ];
+
+  LIBCLANG_PATH = "${libclang.lib}/lib";
```

Comment:
Are you sure this is needed?

Reply:
Yes it throws the following without it. Let me know if there's another option though!

```
 Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"
```

Reply:
Does using `rustPlatform.bindgenHook` work?

Reply:
That worked! Updated